### PR TITLE
Changeset with changes for TPC-H 1G Statistical Data

### PR DESCRIPTION
**Pull request by TheWanderingMinstrel**
This is a revised version of the TPC-H 1G branch. I  believe it has everything that was requested. If not,, let me know. 

++++++++++

This changeset contains two files that have the statistics data
for the TPC-H 1G table data set and modifications necessary to
successfully read in this data. It also contains modifications
to the TPC-H unit tests to test reading and using this data.

Finally, this changeset contains the "expected" results files
for the RunFolderAndVerify unit test for TPC-H 1G data.

### CODE REVIEW

File: test/UnitTest.cs [hunk 1]
```diff
@@ -227,15 +228,9 @@ public void TestBenchmarks()
             TestTpcdsPlanOnly();
             TestTpcdsWithData();
 
+            Tpch.CreateTables();
             TestTpchPlanOnly();
-            TestTpch();
```
Comment (zhouqingqing) [inline test/UnitTest.cs:231]: why do you remove previous tests? Your tpch1g with 1g real stats against empty tables shall be a new test?

Comment (TheWanderingMinstrel) [reply test/UnitTest.cs:231]: Ok... so there are several things going on here...

**Here's what I did:**

1. I renamed TestTpch to TestTpchWithData. Same test, just renamed for symmetry with TestTpcdsWithData.

2.  TestTpchPlanOnly, is a a mirror image of TestTpcdsPlanOnly with empty tables and 1g real stats. It also contains a setup to run RunFolderAndVerify.  

3. I  took RunFolderAndVerify out of TestBenchmarks because while working 1g, it seemed to be redundant. 


**Here's what I think you wanted:**

1. OK to rename TestTpch to TestTpchWithdata, but not necessary

2. Add TestTpchPlanOnly as a mirror of TestTpcdsPlanOnly and to run empty Tpch tables against some other stats file?

3.  Add TestTpch1g as a mirror image of TestTpcdsPlanOnly to run empty tpch tables against 1g stats and add RunFolderAndVerify to the end of this test to verify plans. 

4. Leave the existing  call  to RunFolderAndVerify at the end of BestBenchmarks. 


Is this correct??

Comment (zhouqingqing) [reply test/UnitTest.cs:231]: too complicated ...

We want one more tests loading 1g stats but running against empty tables. The generated plan shall be verified against "expected" results. Do not remove old tests. Of course, refactoring to avoid code duplication is good.

Is this what implemented?

Comment (TheWanderingMinstrel) [reply test/UnitTest.cs:231]: So as things stand now, TestTpchPlanOnly is the name I have given to TestTpch1g. The plan in this test is generated against 1G stats and verified against expected results. 

The only other difference is that that RunFolderAndVerify is no longer at the end of  TestBenchMarks. 


Do  you want me to rename my new test to TestTpch1g and leave the current TestTpchPlanOnly?

Do you want RunFolderAndVerify called from my new test and from TestBenchmarks?
